### DISCUSSION TITLE
Multiple git in header

### DISCUSSION THREAD

**Discussion by countnazgul** [squidfunk/mkdocs-material#5009] [Q&A]
Hi all,

on my docs site im hosting info about multiple projects. And each project have its own github repo.

Is it possible to display the git repository stats (the one in the header) but per project/folder? 

As far as i can see at the moment the only way to set this is via `repo_url` in `mkdocs.yml` but this will be for the whole site.

Thanks!
Stefan

**Comment by squidfunk** [comment]
Jup, that should be possible. You could just hardcode the second git repository in the [`source.html`](https://github.com/squidfunk/mkdocs-material/blob/master/src/partials/source.html) partial – the value of `href` is what Material for MkDocs will use for querying the data from GitHub or GitLab. Something like:

``` jinja2
<!-- Repo from mkdocs.yml -->
<a
  href="{{ config.repo_url }}"
  title="{{ lang.t('source') }}"
  class="md-source"
  data-md-component="source"
>
  <div class="md-source__icon md-icon">
    {% set icon = config.theme.icon.repo or "fontawesome/brands/git-alt" %}
    {% include ".icons/" ~ icon ~ ".svg" %}
  </div>
  <div class="md-source__repository">
    {{ config.repo_name }}
  </div>
</a>

<!-- Second repository (hardcoded) -->
<a
  href="https://github.com/squidfunk/iframe-worker"
  title="{{ lang.t('source') }}"
  class="md-source"
  data-md-component="source"
>
  <div class="md-source__icon md-icon">
    {% set icon = config.theme.icon.repo or "fontawesome/brands/git-alt" %}
    {% include ".icons/" ~ icon ~ ".svg" %}
  </div>
  <div class="md-source__repository">
    squidfunk/iframe-worker 
  </div>
</a>
```

Otherwise, you can put arbitrary keys in the [`extra`](https://www.mkdocs.org/user-guide/configuration/#extra) namespace in `mkdocs.yml` and make it configurable.
